lib/msal-browser/docs/errors.md

```diff
@@ -120,6 +121,26 @@ If you are unable to figure out why this error is being thrown please [open an i
 - Refresh the page. Does the error go away?
 - Open your application in a new tab. Does the error go away?
 
+### block_iframe_reload
+
+**Error Message**: Request was blocked inside an iframe because MSAL detected an authentication response.
+
+This error is thrown when calling `ssoSilent` or `acquireTokenSilent` and the page used as your `redirectUri` is attempting to invoke a login or acquireToken function.
+Our recommended mitigation for this is to set your `redirectUri` to a blank page that does not implement MSAL when invoking silent APIs. This will also have the added benefit of improving performance as the hidden iframe doesn't need to render your page.
+
+✔️ You can do this on a per request basis, for example:
+
+```javascript
+msalInstance.acquireTokenSilent({
+    scopes: ["User.Read"],
+    redirectUri: "http://localhost:3000/blank"
```

Comment:
I would suggest including the html extension to make it clear what this route should be:
```suggestion
    redirectUri: "http://localhost:3000/blank.html"
```